IAMOTZ/ievents
# #152917439 Create template for user dashboard

#### What does this PR do?
Add a template page for users dashboard
#### Description of Task to be completed?
Create template page for user dashboard
#### How should this be manually tested?
- clone the repo
- cd into the template directory
- open the userDashboard.html with any browser of your choice
- the template should open in that browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152917439
#### Screenshots (if appropriate)
![just-now4](https://user-images.githubusercontent.com/26048536/32896208-b8c89a2a-cae2-11e7-92d1-e456832d1097.png)
#### Questions:
None
